test_data/provider_defs/chronicledb.json

```diff
@@ -0,0 +1,11 @@
+{
+  "type": "EdrDataProviderDefinition",
+  "id": "2579a45e-a2c3-4fdb-946a-d98d9d0eacc4",
+  "name": "ChronicleDB Demo",
```

Comment:
If I'm not in the uni network, I get the following error:

```json
{"error":"Reqwest","message":"Reqwest: error decoding response body: expected value at line 1 column 1"}
```

Is it possible to have a better one?

Reply:
3118aa14cf6a425677ffd9fc0c97b096ea549622

Reply:
You committed test data, but I guess we need to have different ones as https://ogcie.iblsoft.com/ states:

> Any data or products exposed by this system are designed for testing or development purposes and SHALL NOT be used for neither operational nor commercial purposes.

Maybe you can sufficiently anonymize it.

Reply:
[47f8a74](https://github.com/geo-engine/geoengine/pull/813/commits/47f8a74066bd0c3751a9e3f192fcedf4085ded73)